src/parser.cc

```diff
@@ -1100,6 +1131,12 @@ bool readXml(tinyxml2::XMLElement *_xml, ElementPtr _sdf,
         std::string uri;
         tinyxml2::XMLElement *uriElement = elemXml->FirstChildElement("uri");
 
+        includeElemIndex++;
```

Comment:
nit: use pre-increment `++i` since it increments the value of the object whereas post-increment `i++` creates a copy then increments

```suggestion
        ++includeElemIndex;
```

Reply:
https://github.com/osrf/sdformat/pull/548/commits/c6d67e086f31c94003c441ffb9a69af2178be2cf